kernel-version.mk

```diff
@@ -66,4 +66,7 @@ KERNEL_DOCKER_TAG = $(KERNEL_BRANCH)-$(KERNEL_COMMIT)-$(KERNEL_COMPILER)
 # one can override the whole tag from the command line and set it to
 # output of make -f Makefile.eve docker-tag-${KERNEL_COMPILER} in github.com/lf-edge/eve-kernel
 KERNEL_TAG ?= docker.io/lfedge/eve-kernel:$(KERNEL_DOCKER_TAG)
-$(info Using kernel from KERNEL_TAG=$(KERNEL_TAG))
+
+#.PHONY: kernel_info
+kernel_info:
```

Comment:
I would suggest to move this target to Makefile, along with all other .phony targets, would be good to also include a description in the help target....

Reply:
We can. Do we need to? This `kernel-version.mk` has all of the kernel stuff.

Reply:
Well, they don't define any target, just variables while we consume KERNEL_TAG in the main Makefile.... but TBH, do we need this kernel_info target at all?

Reply:
Honestly? Probably not. It was getting in the way. I assumed _someone_ needed it, so I left it there, but turned it into a target, rather than always printing it (and breaking things).

I am fine removing it. If someone needs it, they can put it back.

Reply:
Done. Removed.